scripts/SANS/isis_reduction_steps.py

```diff
@@ -1758,11 +1766,12 @@ def _evaluate_settings(self):
 
         # Check that for each of these settings we only have one run number specified, else raise an error
         has_max_one_run_number = lambda indices : len(set([run_number[index] for index in indices])) < 2
-        if (not has_max_one_run_number(indices_time_detector) or
-            not has_max_one_run_number(indices_time_monitor) or
-            not has_max_one_run_number(indices_uamp_detector) or
-            not has_max_one_run_number(indices_uamp_monitor) ) :
-            raise RuntimeError("DarkRunSubtraction: More background correction runs have been specified than are allowed. "
+        if not has_max_one_run_number(indices_time_detector) or \
+           not has_max_one_run_number(indices_time_monitor) or \
+           not has_max_one_run_number(indices_uamp_detector) or \
+           not has_max_one_run_number(indices_uamp_monitor):
+            raise RuntimeError("DarkRunSubtraction: More background correction runs have been "
```

Comment:
Was that a pylint warning? Braces are a valid way of having multi-line expressions in python. But if pylint says that the `\` approach is preferred, then that is fine.